### DISCUSSION TITLE
Taiko with mocha in a docker

### DISCUSSION THREAD

**Discussion by zaalgol** [getgauge/taiko#2019] [Development]
How i could run Taiko with mocha in a docker?

There is only documentation for taiko with mocha, or taiko with getgauge in a docker.

**Comment by zabil** [comment]
You can re-use the instructions for docker without steps for installing gauge. Mocha should be a part of your `package.json` so you need to

```
npm install
npm test // make sure the test command in package.json runs mocha
```

**Reply by saikrishna321** [reply]
https://github.com/saikrishna321/taiko-docker
